SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -2198,6 +2198,29 @@ class CFVMFlowSolverBase : public CSolver {
       Inlet_FlowDir[val_marker][val_vertex][val_dim] = val_flowdir;
   }
 
+  /*!
+   * \brief Set a component of the farfield velocity vector.
+   */
+  inline void UpdateFarfield_Velocity(const CConfig* config) final {
+    // /*--- Retrieve the AoA and AoS (degrees) ---*/
+    su2double AoA = config->GetAoA() * PI_NUMBER / 180.0;
+    const su2double AoS = config->GetAoS() * PI_NUMBER / 180.0;
+    /*--- Update the freestream velocity vector at the farfield
+     * Compute the new freestream velocity with the updated AoA,
+     * "Velocity_Inf" is shared with config. ---*/
+
+    const su2double Vel_Infty_Mag = GeometryToolbox::Norm(nDim, Velocity_Inf);
```

Comment:
Find a way of sharing this code with the function I pointed you to please.

Reply:
Is it OK to remove duplication and use this call in the function?